public/pages/VisualCreatePolicy/containers/VisualCreatePolicy/VisualCreatePolicy.tsx

```diff
@@ -292,25 +293,30 @@ export class VisualCreatePolicy extends Component<VisualCreatePolicyProps, Visua
 
     const descriptionData = [
       {
-        renderComponent: (
-          <EuiText size="s" color="subdued">
-            Policies let you automatically perform administrative operations on indices.{" "}
-            <EuiLink href={POLICY_DOCUMENTATION_URL} target="_blank" rel="noopener noreferrer">
-              Learn more
-            </EuiLink>
-          </EuiText>
-        ),
-      },
+        description: "Policies let you automatically perform administrative operations on indices.",
+        links: {
+          label: "Learn more",
+          href: POLICY_DOCUMENTATION_URL,
+          iconType: "popout",
+          iconSide: "right",
+          controlType: "link",
+          target: "_blank",
+          flush: "both",
+        } as TopNavControlLinkData,
+      } as TopNavControlDescriptionData,
     ];
+
     const padding_style = useNewUX ? { padding: "0px 0px" } : { padding: "25px 50px" };
     return (
       <div style={padding_style}>
         {!useNewUX ? (
           <>
-            <EuiTitle size="l">
-              <h1>{isEdit ? "Edit" : "Create"} policy</h1>
```

Comment:
Does the heading need to be rendered at a specific size (usually heading sizes are just `<h2>` wrapped under `<EuiText size="s">`)

Reply:
This is page heading for old UI